### ISSUE TITLE
Move Epsilon Version Stuff To Twisted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11200]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1663|
|Type|enhancement|
|Created|2006-04-24 00:07:25Z|

HIGH PRIORITY

Oh right, and use it everywhere.

<details><summary>Searchable metadata</summary>

```
trac-id__1663 1663
type__enhancement enhancement
reporter__radix radix
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1145837245000000 1145837245000000
changetime__1145845008000000 1145845008000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [16667]) Move epsilon.versions to twisted.python.versions, and use it.

Author: radix
Reviewer: itamarst
Fixes [#11200](https://github.com/twisted/twisted/issues/11200)

All "Projects" have a 'version' attribute, which is an instance of
twisted.python.versions.Version. They also have a __version__ attribute which
is a string version number (maybe including an SVN revision). 

TODO: Update any release automation that this affects.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
